sdk/communication/Azure.Communication.CallingServer/src/Models/ChannelAffinity.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallingServer
+{
+    /// <summary>
+    /// Channel affinity for a participant.
+    /// </summary>
+    public class ChannelAffinity
+    {
+        /// <summary> Channel number to which bitstream from a particular participant will be written. </summary>
+        public int? Channel { get; set; }
```

Comment:
does this mean that Channel is optional? what is the use of this, if channel is optional?

Reply:
Good catch, will make it non-nullable